sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -48,7 +48,10 @@ public Context start(String spanName, Context context) {
 
         Builder spanBuilder = getSpanBuilder(spanName, context);
         Span span = spanBuilder.startSpan();
-
+        if (span.isRecording()) {
+            span.setAttribute(AZ_NAMESPACE_KEY,
+                AttributeValue.stringAttributeValue(HttpTraceUtil.parseNamespaceProvider(spanName)));
```

Comment:
Should this add the attribute if the parsed name is `""`? It appear in the policy it doesn't.

Reply:
I think that is what we are following for other span attributes as well.

Reply:
updated it to not have the span attribute if it does not exist.